### DISCUSSION TITLE
Welcome to Retrospective Extension Discussion

### DISCUSSION THREAD

**Discussion by polatengin** [microsoft/vsts-extension-retrospectives#110] [Show and tell]
👋 Welcome!
---
We’re using Discussions as a place to connect with other users of the extension.

We hope that you:

* Ask questions you're wondering about.
* Share ideas.
* Engage with other users.
* Welcome others and are open-minded.

Remember that this is an extension we build together 💪.

**Comment by marktold** [comment]
Can it be that currently I can enter only on letter at the time?

**Comment by karen-ens** [comment]
Can I ask what the purpose of the blurred feedback is on the Collect tab when it is visible on the other tabs?

We would like to make feedback private until we decide when it is visible (controlled by admin of the project).

**Comment by ghost** [comment]
I think this tool is great. Some feature requests

- Support live updates
- Combine the features in all 4 tabs to a single view
- Drag and drop to move cards like on a sprint taskboard
- Can vote for individual items in a group instead of the whole group

**Reply by Neil-Craig-Nintex** [reply]
+1 on the live updates request

**Comment by DimitryOlshansky** [comment]
Hi,
We like the tool and already used it on 6 retrospectives,
we would be grateful if you could add filter by team member. it could help when focusing on person by person items.

**Comment by ZoeDee990** [comment]
I see that "Measure Team Effectiveness" allows input, but where can I see the collective results?

**Comment by WilliamMakaika** [comment]
Hi is there a way to set up auto notification when retrospectives have been done?

**Comment by babuvaka1** [comment]
History Tab not loading Azure Devops 2020, On-Prem, retrospective extension version: 1.70.0.6

**Comment by gneubauer** [comment]
Hi, is it possible to tailor team assessment ? or can only be fixed.
